test/utils-test.ts

```diff
@@ -30,4 +30,10 @@ describe('WSL path format to Windows', () => {
     const windows = 'D:\\Users\\user1\\AppData';
     assert.equal(toWinDirFormat(wsl), windows);
   });
+
+  it('getLocalAppDataPath returns a correct path', () => {
+    process.env.PATH = '/mnt/c/Users/user1/.bin:/mnt/c/Users/user1/AppData/';
```

Comment:
let's be sure to put path back to normal after this or better yet make this function accept it as input :)

Reply:
yeah much better to have it as a param